database/migrations/2021_05_31_164031_extend_room_id.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class ExtendRoomId extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('rooms', function (Blueprint $table) {
+            $table->string('id',15)->change();
+        });
+        Schema::table('meetings', function (Blueprint $table) {
+            $table->string('room_id',15)->change();
+        });
+        Schema::table('room_files', function (Blueprint $table) {
+            $table->string('room_id',15)->change();
+        });
+        Schema::table('room_user', function (Blueprint $table) {
+            $table->string('room_id',15)->change();
+        });
+    }
+
+    /**
+     * Reverse the migrations.
+     *
+     * @return void
+     */
+    public function down()
+    {
+        Schema::table('rooms', function (Blueprint $table) {
```

Comment:
The rollback is throwing the error 

`Cannot change column 'id': used in a foreign key constraint 'meetings_room_id_foreign' of table 'pilos.meetings'`

Reply:
The question is, whether a rollback is possible or not? I can fix this bug but in case if there are rooms with longer ids in the database (e.g. imported from greenlight) they will be cutted and then it wouldn't work!

Reply:
@SamuelWei see commit 64c845c. But as I said, this migration can't be rolled back if there are longer ids!

Reply:
@dennis95stumm I'm fine with this solution, just wanted to make sure the immediate rollback after a migration is possible